base/utils/R/utils.R

```diff
@@ -512,7 +512,7 @@ temp.settings <- function(settings.txt) {
 ##' @author David LeBauer
 tryl <- function(FUN) {
   out <- tryCatch(FUN, error = function(e) e)
-  ans <- !any(class(out) == "error")
+  ans <- !any(inherits(out, "error"))
```

Comment:
```suggestion
  ans <- !inherits(out, "error")
```

inherits returns a single logical value, so it removes the need for `any` as well